securedrop_client/gui/actions.py

```diff
@@ -0,0 +1,69 @@
+"""
+The actions available to the journalist.
+
+Over time, this module could become the interface between
+the GUI and the controller.
+
+Copyright (C) 2021 The Freedom of the Press Foundation.
```

Comment:
see https://github.com/freedomofpress/securedrop-client/issues/1443

Reply:
I removed them like I did in the SecureDrop SDK some time ago. Whichever convention we end up adopting I'm not worried about these files, they'll be easy to track and update as needed.